examples/purchase-tester/src/main/java/com/revenuecat/purchasetester/PackageCardAdapter.kt

```diff
@@ -48,19 +79,57 @@ class PackageCardAdapter(
                 currentPackage.packageType.toString()
             }
 
-            binding.packageDetailsJsonObject.detail = ""
-                // TODOBC5 currentPackage.product.originalJson.toString(JSON_FORMATTER_INDENT_SPACES)
+            binding.packageDetailsJsonObject.detail = product.googleProduct?.productDetails?.toString() ?: "TODO Amazon"
+
+            bindPurchaseOptions(product)
 
             binding.root.setOnClickListener {
                 with(binding.packageDetailsContainer) {
                     visibility = if (visibility == View.GONE) View.VISIBLE else View.GONE
                 }
             }
         }
+
+        @SuppressLint("SetTextI18n")
+        private fun bindPurchaseOptions(product: StoreProduct) {
+            binding.packagePurchaseOptionGroup.removeAllViews()
+            val numberOfPurchaseOptions = product.purchaseOptions.size
+            product.purchaseOptions.forEach { purchaseOption ->
+                val radioButton = RadioButton(binding.root.context).apply {
+                    text = purchaseOption.toButtonString()
+                    tag = purchaseOption
+                    if (numberOfPurchaseOptions == 1) isChecked = true
```

Comment:
This is a convenience so we don't need to tap on the option when there is only one. I thought about automatically selecting one when there are more than one... But I think it's better to make the user choose one.